prow/plugins/projectmanager/projectmanager.go

```diff
@@ -108,61 +115,102 @@ type githubClient interface {
 	CreateProjectCard(columnID int, projectCard github.ProjectCard) (*github.ProjectCard, error)
 }
 
-func handlePR(gc githubClient, projectManager plugins.ProjectManager, log *logrus.Entry, pe github.PullRequestEvent) error {
-	// Only respond to label add or issue/PR open events
-	if pe.Action != github.PullRequestActionOpened &&
-		pe.Action != github.PullRequestActionReopened &&
-		pe.Action != github.PullRequestActionLabeled {
+type eventData struct {
+	id     int
+	number int
+	isPR   bool
+	org    string
+	repo   string
+	state  string
+	labels []github.Label
+	remove bool
+}
+
+type DuplicateCard struct {
+	projectName string
+	issueURL    string
+}
+
+func (m *DuplicateCard) Error() string {
+	return fmt.Sprintf(issueAlreadyInProject, m.issueURL, m.projectName)
+}
+
+func handleIssueRequest(pc plugins.Agent, ie github.IssueEvent) error {
+	if !handleIssueActions[ie.Action] {
+		return nil
+	}
+	eventData := eventData{ie.Issue.ID, ie.Issue.Number, false, ie.Repo.Owner.Login, ie.Repo.Name, ie.Issue.State, ie.Issue.Labels, false}
+	if ie.Action == github.IssueActionUnlabeled {
+		eventData.remove = true
```

Comment:
`eventData.remove = ie.Action == github.IssueActionUnlabeled` (or just use `ie.Action == github.IssueActionUnlabeled` instead of `false` in the literal definition)  and similar for the `handlePullRequest`.

Reply:
Fixed.